options.html

```diff
@@ -192,18 +192,25 @@ <h1>Town Tune Editor</h1>
 					<section>
 						<p>Compose a tune to be played every hour on the hour</p>
 						<div class="pitch-template hidden">
-							<input class="pitch-name" type="text" maxlength="2" size="2">
-							<input class="pitch-slider" type="range" min="0" max="14" step="1" orient="vertical">
+							<input class="pitch-name"   type="text"  maxlength="2" size="2">
+							<input class="pitch-slider" type="range" min="0" max="16" step="1" orient="vertical">
 						</div>
 						<div class="staff staff1"></div>
 						<div class="staff staff2"></div>
+						<br>
+						<p>Town tune volume:</p>
+						<input id="townTuneVolume" type="range" min="0" max="1" step="0.01">
+						<span  id="townTuneVolumeText"></span> 
 					</section>
-
+					
 					<header>
 						<div style="padding: 0; margin: 0; height:18px;"></div>
 						<div class="controls">
 							<button class="play-tune">Play!</button>
 							<div class="spacer"></div>
+							<button class="reset-tune">Reset</button>
```

Comment:
I agree that changing it to "Set Default" could clear up some confusion, though it may also imply that whatever tune they have, is set as the default, for something. Maybe "Reset to Default"?

A "Discard" button could also be a good addition, however it may needlessly complicate the town tune editor, especially if we also keep the "Reset to Default" button, making the page cluttered with buttons. I also think that it's rather clear to any user that in order to not change their town tune, they just have to not click "Save", so while it could be a decent addition, I feel it would over-clutter or over-complicate the editor.

Reply:
The button text has been changed to "Reset to Default"